airbyte-featureflag/src/main/kotlin/Flags.kt

```diff
@@ -19,6 +19,8 @@ object ApplyFieldSelection : EnvVar(envVar = "APPLY_FIELD_SELECTION")
 
 object PerfBackgroundJsonValidation : Temporary(key = "performance.backgroundJsonSchemaValidation")
 
+object StrictComparisonNormalizationEnabled : Temporary(key = "normalization.strictComparisonNormalizationEnabled")
```

Comment:
I think that you're missing an update in the file flags.yml in order to be able to use this flag in Open source. If the plan is to remove the other flag, we should add it.

Reply:
is flags.yml just default values to handle the case where OSS users can't reach launchdarkly?

also, which other flag are you referring to? (the env var for image tag, or the env var for enabling strict_comparison)

Reply:
added an entry to flags.yml, ptal just in case and then I'll mark as ready to ship

Reply:
yes, it is about having a configurable value in OSS.

Reply:
If you want to test it locally, you'll need to add the path to the flag file in the application configuration like what is done in https://github.com/airbytehq/airbyte/pull/22397/files#diff-d0c67bf55b2b8412bf233bf144f0a4f74f60bb749faccbe0aba89c77730d12ba

Reply:
tested locally with that setup, confirmed that normalization runs with `strict_comparison` after flipping `flags.yml` to true